cvprac/cvp_api.py

```diff
@@ -85,13 +85,13 @@ def __init__(self, clnt, request_timeout=30):
         self.log = clnt.log
         self.request_timeout = request_timeout
 
-    def check_v7(self, msg):
-        ''' Checking the version is above CVP 2021.3.0
+    def _check_version_greater_or_equal(self, version, msg):
+        ''' Checking the version greater than or equal to provided API version
```

Comment:
This is the inverse of what the function does. Changing to function name to Less Than implies that if I send a version that is less than the current API version the function would return True. Greater or Equal means that if the provided API version is greater or equal it will return True, which is what the function does.

Reply:
ahh ok this compare function using the operator library is very interesting. I'll think about this format a bit.

Reply:
I've updated the version comparison to use the operator library. Let me know what you think.